apis/outlook/calendar/code/main.go

```diff
@@ -0,0 +1,143 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/gptscript-ai/tools/apis/outlook/calendar/code/pkg/commands"
+	"github.com/gptscript-ai/tools/apis/outlook/calendar/code/pkg/graph"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("Usage: calendar <command>")
+		os.Exit(1)
+	}
+
+	command := os.Args[1]
+
+	switch command {
+	case "listCalendars":
+		if err := commands.ListCalendars(context.Background()); err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+	case "listEventsToday":
+		if err := commands.ListEventsToday(context.Background()); err != nil {
```

Comment:
Does this have to be a separate function? Can we use `commands.ListEvents` with a relevant `start` and `end`?

Reply:
Yeah it was silly of me to implement it the way I did lol

Reply:
Fixed